.github/workflows/module_replacements.yml

```
@@ -0,0 +1,75 @@
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git add .
+          git commit -m "chore: update e18e module replacements data"
+          git push origin HEAD:chore/update-module-replacements-data -f
+          PR_URL=$(gh pr create --title "chore: update e18e module replacements data" --body "Automated update for module replacements data." --base main)
+          echo "pr_url=$PR_URL" >> $GITHUB_OUTPUT
+
+      - name: Auto merge
+        if: ${{ steps.create-pull-request.outputs.pr_url != '' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] property "create_pr" is not defined in object type {changes: {conclusion: string; outcome: string; outputs: {string => string}}; create-pull-request: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)